launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
I place the cylinder on top of the sensor and I modified the vito-robot/vito_description/model/cylinder.urdf.xacro using this code to have a realistic measure:

<gazebo reference="cylinder"> 
		selfCollide>true</selfCollide
		<gravity>false</gravity> 
		<mu1>0.6</mu1>
		 <mu2>0.6</mu2> 
		 <kp>10000.0</kp>
		 <kd>100.0</kd> 
	 </gazebo>

without this FT sensor measure swings between two values, which does't have physical sense.